esp_hosted_ng/host/sdio/esp_sdio.c

```diff
@@ -866,27 +878,35 @@ static const struct dev_pm_ops esp_pm_ops = {
 	.resume = esp_resume,
 };
 
+static const struct of_device_id esp32_sdio_of_match[] = {
+	{ .compatible = "espressif,esp32_sdio", },
```

Comment:
change esp32_sdio to esp_sdio. this will rule out any confusion for esp32 and esp32c6 chipsets, wherever applicable